tests/protos/protos/TestRectangleArena.proto

```diff
@@ -0,0 +1,107 @@
+#VRML_SIM R2020b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# tags: static
+# A configurable rectangle arena composed of a floor surrounded by walls.
+
+PROTO TestRectangleArena [
+  field SFVec3f    translation     0 0 0
+  field SFRotation rotation        0 1 0 0
+  field SFString   name            "rectangle arena"
+  field SFString   contactMaterial "default"                       # Is `Solid.contactMaterial`.
+  field SFVec2f    floorSize       1 1                             # Defines the size of the floor.
+  field SFVec2f    floorTileSize   0.5 0.5                         # Defines the size of the texture used for the floor.
+  field SFNode     floorAppearance Parquetry { type "chequered" }  # Defines the appearance of the floor.
+  field SFFloat    wallThickness   0.01                            # Defines the thickness of the walls.
+  field SFFloat    wallHeight      0.1                             # Defines the height of the walls.
+  field SFNode     wallAppearance  BrushedAluminium {}             # Defines the appearance of the wall.
+
+  # Deprecated in R2019a
+  hiddenField MFString   floorTextureUrl "textures/checkered_parquetry.jpg"
+  hiddenField MFString   wallTextureUrl  "textures/metal.jpg"
+  # Deprecated in R2019a-rev2
+  hiddenField SFColor    wallColor 0.8 0.8 0.8
+  hiddenField SFVec2f    wallTileSize 0.5 0.5
```

Comment:
Can't we remove these deprecated fields? It seems they are not useful in this test.

Reply:
Yes, I will remove them